spec/features/items/index_spec.rb

```diff
@@ -63,8 +63,10 @@
 
       flying_disc = @brian.items.create(name: "A Flying Disc", description: "Flies for days!", price: 10, image: "https://hw.menardc.com/main/items/media/CEGEN001/ProductLarge/253-0107_P_4.jpg", inventory: 32)
 
-      order = Order.create!(name: "name", address: "address", city: "city", state: "state", zip: 80210)
-      order_2 = Order.create!(name: "name", address: "address", city: "city", state: "state", zip: 80210)
+      user = create(:user)
```

Comment:
This is created using FactoryBot, right? Love it.

Reply:
yup!